Kickstarter-iOS/Views/SelectCurrencyTableViewHeader.swift

```diff
@@ -0,0 +1,63 @@
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+final class SelectCurrencyTableViewHeader: UITableViewHeaderFooterView {
```

Comment:
I've had a quick peek at the documentation and am starting to think we should have used `UIView` subclass instead. It looks like `UITableViewHeaderFooterView` is designed only for the `section` headers/footers that take advantage of the reuse of those views (just like cells). On the other hand `UITableView`'s `tableHeaderView`/`tableFooterView` is an accessory view that isn't reused.

https://developer.apple.com/documentation/uikit/uitableviewheaderfooterview

Reply:
ok good point - I think we were trying a bunch of things to get it to layout properly and may have got a false positive on this change 👍